.changeset/tricky-dragons-explain.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': patch
+---
+
+Small polish to the dev overlay:
+- Links inside the Astro panel now open in a new tab
+- The "Copy debug info" button will now reset back to its original state after 3.5 seconds without needing to toggle off the plugin
```

Comment:
Changelogs for prereleases are always a tricky balance... do we need to highlight all of these in detail? Maybe something more like "fixes a number of small user experience bugs with the dev overlay"